docs/report/report.tex

```diff
@@ -731,24 +731,75 @@ \subsubsection*{Meilenstein 4} $~$ \\
 Um insgesamt auch die Arbeit kontinuierlicher zu gestalten, sollten Aufgaben in kleinere Probleme aufgeteilt werden.
 
 \subsubsection*{Meilenstein 5} $~$ \\
-Der Meilenstein fünf vom 14.01.2021 bis zum 28.01.2021 sollte die Entwicklung am Produkt weitgehend zu abschließen, sodass sich das Team auf den Bericht fokussieren konnte.
+Mit dem fünften Meilenstein sollte zwischen dem 14.01.2021 und 28.01.2021 die Entwicklung am Produkt weitgehend abgeschlossen werden, sodass der Fokus mehr auf den Bericht gerichtet werden konnte.
+
 %Ziel
-Geplant war, die letzten Anwendungszenarien final umzusetzen, sodass die letzten beiden Meilensteine höchstens zum Beheben kleinerer Fehler benötigt werden würden.
+Geplant war hierbei, die letzten Anwendungszenarien final umzusetzen, sodass die letzten beiden Meilensteine höchstens zum Beheben kleinerer Fehler benötigt genutzt werden müsste.
 Somit war das Ziel die Umsetzung einer Oberfläche für die Einstellungen der \textit{Leek Box} selbst als auch der einzelnen Tags.
 Die User Interface Elemente sollten, sofern noch nicht geschehen, ein einheitliches Design erhalten.
 Auch sollte die Dokumentation zur Bedienung der \textit{Leek Box} fertiggestellt sein und die Arbeit für den Bericht begonnen werden.
+
+% Durchführung
+Die Einstellungen der \textit{Leek Box} wurde hinzugefügt und wurde durch ein Zahnrad auf der Startseite erreichbar.
+In diesen Einstellungen konnten die Nutzer nun den zu verwendenden Lautsprecher für die Wiedergabe auswählen und sich ausloggen.
+\paragraph*{Flow-Charts} $~$ \\
+\begin{wrapfigure}{r}{0cm}
+  \includegraphics[width=0.45\textwidth]{Settings.jpeg}
+  \caption{Leek Box Einstellungen}
+\end{wrapfigure}
+Auch war nun das Bearbeiten der Tags möglich.
+Dabei handelte es sich lediglich um die beim Anlegen eingestellten Informationen: Name, Lied und Bild des Tags.
+\paragraph*{Flow-Charts} $~$ \\
+\begin{wrapfigure}{r}{0cm}
+  \includegraphics[width=0.45\textwidth]{TagDetails.jpeg}
+  \caption{Tag Einstellungen und Details}
+\end{wrapfigure}
+An dieser Stelle war es erneut sehr praktisch ein Framework wie \textit{Vue} zu nutzen, da somit die bereits vorhandenen \textit{views} wiederverwendet werden konnten.
```

Comment:
```suggestion
An dieser Stelle zeigte sich erneut die Vorzüge von Frameworks wie  \textit{Vue}, da somit die bereits vorhandenen \textit{components} wiederverwendet werden konnten.
```

Auch hier gerne nochmal erwähnen, dass es die View TagDetails ist, in der die Views geladen werden. Da ergeben sich eigentlich einige erwähnenswerte Technische Punkte, wenn man sich mal den Code anguckt.

Reply:
was meinst du?
Die Child parent kommunikation?

Reply:
Ne es gibt ja die View TagDetails.vue. Da werden ja die components von AddTag teilweise wiederverwendet. Da könnte man sich ja die TagDeatils View mal angucken - Also den Code und die mal beschrieben und ein bisschen technisch (vgl. Milesteon 4)

Reply:
```suggestion
An dieser Stelle zeigte sich erneut die Vorzüge von Frameworks wie  \textit{Vue}, da somit die bereits vorhandenen \textit{components} wiederverwendet werden konnten.
\colorbox{red}{ausformulieren}
```